src/main/java/com/jvm_bloggers/core/newsletter_issues/domain/NewsletterIssue.java

```diff
@@ -37,7 +36,7 @@
 @Builder
 @Getter
 @ToString(exclude = {"blogPosts", "newBlogs"})
-public class NewsletterIssue {
+public class NewsletterIssue implements NewsletterIssueBaseData {
```

Comment:
I would prefer to have a DTO with only required data than an interface that pollutes domain object.